server/index.js

```diff
@@ -439,7 +440,7 @@ app.post('/get_client_urls', function (req, res) {
                 return;
             });
         };
-        
+
         verify_password(body.session, body.password, success, fail);
```

Comment:
@frederickjansen I'm wondering if it's easier to keep everything in lowercase consistently, although I could see how the uppercase letters are a little easier to read. I'll keep it uppercase on the client side for now.